canonical/canonical.com
# WD-1154 Get hiring leads bio

Reverts canonical/canonical.com#873

## QA

Use these links to check:
https://pastebin.canonical.com/p/vRcskFFyzt/

- First link is an actual application
- Second link is a dummy application for testing the only video we have, as we I don't have an application of that specific hiring lead.

Make sure [directory API works](https://chat.canonical.com/canonical/pl/4gewjhq9abb68fctx8heb5f4bw).
1. Login into the prod-comms k8s cluster
2. Find a pod
```
kubectl get pods | grep "canonical"
kubect -ti exec <canonical-com-pod-name> -- /bin/bash
```
Once inside of the pod, install curl if needed
```
apt-get update && apt-get install curl
curl -I -k https://directory.wpe.internal
```
It should return 200

- Directory API is currently down, so if you check https://pastebin.canonical.com/p/HqC8VbRCnG/ it should should the application page without bio or avatar.
